nvtabular/framework_utils/tensorflow/layers/embedding.py

```diff
@@ -307,10 +359,11 @@ def call(self, inputs):
         numeric_inputs = []
         for feature_column in self.feature_columns:
             if isinstance(feature_column, fc.NumericColumn):
-                numeric_inputs.append(inputs[feature_column.key])
+                numeric_inputs.append(_handle_continuous_feature(inputs, feature_column))
             else:
                 table = self.embedding_tables[feature_column.key]
-                x = x + tf.gather(table, inputs[feature_column.key][:, 0])
+                embeddings = _categorical_embedding_lookup(table, inputs, feature_column.key, "sum")
```

Comment:
Should "sum" be self.sparse_aggregation here (assuming we set it in the constructor) ? 
```suggestion
                embeddings = _categorical_embedding_lookup(table, inputs, feature_column.key, self.sparse_aggregation)
```

Reply:
This kwarg is removed, just removed it from the documentation too. Should have never been added, linear layers always use sum aggregation by definition